virttest/qemu_vm.py

```diff
@@ -3544,6 +3544,23 @@ def get_spice_var(self, spice_var):
         """
         return self.spice_options.get(spice_var, None)
 
+    def _get_bus(self, params, *args, **kwargs):
+        """
+        Deal with different buses for multi-arch
+        """
+        if self.params.get("machine_type").startswith('s390'):
+            return self._get_ccw_bus()
+        else:
+            nic_params = params
+            pcie = args[-1]
```

Comment:
`nic_params = params` is meaningless, and there is no guarantee that the position of pcie will be in the last position in the future.